.github/workflows/tests.yml

```diff
@@ -523,6 +523,101 @@ jobs:
         TOX_PARALLEL_NO_SPINNER: 1
         PY_COLORS: 0
 
+  mongodb:
+
+    strategy:
+      matrix:
+        test-directory:
+          - datastore_pymongo
+
+    runs-on: ubuntu-latest
+
+    services:
+      mongodb:
+        image: mongo:3.6.4
```

Comment:
Why use mongo:3.6.4 instead of mongo:4 or mongo:4.4?

Reply:
`parallel_scan` was removed in mongo 4, and 3.6.4 is what we were using internally.